spyder/app/tour.py

```diff
@@ -28,10 +28,38 @@
 
 # Local imports
 from spyder.config.base import _, get_image_path
+from spyder.config.main import CONF
+from spyder.config.gui import is_dark_font_color
 from spyder.py3compat import to_binary_string
 from spyder.utils.qthelpers import add_actions, create_action
 from spyder.utils import icon_manager as ima
 
+
+def get_colors():
+    """
+    Get top and background colors for the tour based on the color theme
+    and color scheme currently selected.
+    """
+    ui_theme = CONF.get('color_schemes', 'ui_theme')
+    color_scheme = CONF.get('color_schemes', 'selected')

```

Comment:
Let's define three constants below this line, to use them in this function:

```
dark_color = QColor.fromRgb(35, 38, 41)
white = QColor.fromRgb(255, 255, 255)
grey = QColor.fromRgb(230, 230, 230)
```

to use them below.